jax/experimental/maps.py

```diff
@@ -305,7 +306,7 @@ def make_xmap_callable(fun: lu.WrappedFun,
                                     *in_avals)
   else:
     # We have to trace again, because `f` is a linear function, so we can't just return it.
```

Comment:
Maybe this tweak would make the comment clearer? (This isn't part of this PR, but I just noticed it.)

```suggestion
    # We have to trace again, because `f` is an lu.WrappedFun, so we can't just return it.
```